src/nvim/api/win_config.c

```diff
@@ -330,7 +349,76 @@ static bool parse_float_bufpos(Array bufpos, lpos_T *out)
   return true;
 }
 
-static void parse_border_style(Object style, FloatConfig *fconfig, Error *err)
+static void parse_border_title(Object title, Object title_pos, Object width, FloatConfig *fconfig,
```

Comment:
`Object width` is not needed here`. `title_width` is being calculated from the contents already.